pkg/synthesis/tests_expected_output/ExampleDumbeldore.txt

```diff
@@ -0,0 +1,12 @@
+category: Application
```

Comment:
is there option to override expected output with new output?

Reply:
We can change the file. Or is there something else you had in mind?

Reply:
code that updates those files automatically when needed?

Reply:
Opened an issue 
#104